README.md

```diff
@@ -62,7 +62,11 @@ distance2 = distance1 / 7
 # distance2.km == 2.85
 # Note: rounded to 2 decimals
 
-__lt__, __gt__, __eq__, __le__, __ge__
+__lt__,
+__gt__,
+__eq__,
+__le__,
+__ge__
```

Comment:
Undo this change

Reply:
I didn't catch you)) What exactly to cancel. Add _validate method was before and use import operator?

Reply:
Do as before 
`__lt__, __gt__, __eq__, __le__, __ge__`